spec/routes/api/project/location/postgres_spec.rb

```diff
@@ -334,7 +334,7 @@
       it "not exist ubid in location" do
```

Comment:
Since we centralized this logic with `check_visible_location` helper, we might test `/project/#{project.ubid}/location/foo_location` path once instead of testing it for all resources separately.

Reply:
That makes sense.  I'll push a commit for that.

Reply:
Unfortunately, you cannot test `/project/#{project.ubid}/location/foo_location` separately, since by itself, it isn't a valid path.  We can pick one spec that tests the behavior, and delete the other ones that would be redundant as they test the same behavior.  I pushed a commit for that, as well as one that fixes some related specs.